Sources/Logging/Logging.swift

```diff
@@ -43,6 +43,9 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
+    @available(macOS 10.15, *)
+    @TaskLocal
+    public static var logger: Logger = Logger(label: "NoOp", SwiftLogNoOpLogHandler())
```

Comment:
I'd suggest a name like `current` or `inherited` so that it reads more naturally than `Logger.logger` (which sounds like a factory method that creates a brand new logger instead)

Reply:
I concur with this completely.

Reply:
Agree! `Logger.current` or `Logger.inherited` both sound good to me but open to other stuff too